invenio_rdm_records/stats/templates/events/record_view/os-v2/record-view-v1.0.0.json

```diff
@@ -0,0 +1,70 @@
+{
+  "index_patterns": ["__SEARCH_INDEX_PREFIX__events-stats-record-view-*"],
```

Comment:
silly question: aren't these mappings already defined in invenio stats?

Reply:
You mean the [`contrib` mappings](https://github.com/inveniosoftware/invenio-stats/blob/master/invenio_stats/contrib/record_view/os-v2/record-view-v1.json), right?

Unfortuantely, they don't match our use-case exactly – for example, they're lacking the `parent_recid` mapping, which we're using for statistics of a single record version vs. all vesrions.
I'm not 100% sure about the actual usage of the `contrib` mappings (probably they're purely examples), so I left them as-is.